pkg/types/gateway.go

```diff
@@ -7,6 +7,7 @@ import (
 const (
 	DefaultExtractedObjectPath         string        = "/data/unpacked"
 	DefaultVolumesPath                 string        = "/data/volumes"
+	DefaultExternalVolumesPath         string        = "/data/external-volumes"
```

Comment:
do we need this in gateway? Assuming we aren't allowing users to upload through cli, etc it might not matter?